packages/@lwc/babel-plugin-component/src/__tests__/wire-decorator.spec.js

```diff
@@ -40,6 +40,72 @@ describe('Transform property', () => {
                       },
                       static: {
                         key2: ["fixed", "array"]
+                      },
+                      config: function(host) {
+                        let v1 = host.prop1;
+                        return {
+                          key2: ["fixed", "array"],
+                          key1: v1 != null ? v1 : undefined
```

Comment:
hmm, it is still buggy. if `host.prop1` is `null`, it will return undefined :(, basically, this form should be used if you are accessing something that has at least two levels deep, e.g.: host.prop1.prop2, but for `host.prop1` you don't need thing.

Reply:
my bad, i understood that was the desired behavior.

Since that's case and we need to handle null/undefined, we could remove the condition for the last prop access too, returning `acc.lastProp`, the most common cases will look like this:

```
foo (n = 1): 
   prop: host.foo

foo.bar (n = 2):
   let v = host.foo;
   prop: v != null ? v.bar : undefined

foo.bar.baz (n = 3):
   let v = host.foo;
   prop: v != null &&
        (v = v.bar) != null
          ? v.baz
          : undefined
```